### PR TITLE
#546 #568  Options minor cleanup fixes

### LINKED ISSUE #546: tests_views.py:889-891: Create Product.catalog_name...
The puzzle `498-c440aac6` from #498 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/23a96394014d28ae74e5644014140046b8b56bc9/stroyprombeton/tests/tests_views.py#L889-L891

The puzzle was created by duker33 on 03-Apr-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### LINKED ISSUE #568: series.html:40-41: Rename `category_products.html` to...
The puzzle `560-497606ea` from #560 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/b82ec9976d33465440e90894d130e59c78d989c9/templates/catalog/series.html#L40-L41

The puzzle was created by duker33 on 07-Apr-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #546 
Closes #568

### CODE REVIEW

File: templates/ecommerce/prices/price.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <offers>
       {% for option in options %}
         <offer id="{{ option.product.id }}" available="{{ option.in_stock|yesno:'true,false' }}">
-          <name>{{ option.mark }} - {{ option.product.name }} {{ option.specification }}</name>
+          <name>{{ option.mark }} - {{ option.product.name }}</name>
```
Comment (artemiy312) [inline templates/ecommerce/prices/price.yml]:
```suggestion
          <name>{{ option.catalog_name }}</name>
```

File: templates/ecommerce/order/email.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
         <td style="border-color:#E4E4E4;padding:10px;text-align:left">
           <a href="{% full_url 'product' position.product_id %}"
              style="text-decoration: none"><!--
-             -->{{ position.name }} {{ position.mark }}
+             -->{{ position.catalog_name }}
```
Comment (artemiy312) [inline templates/ecommerce/order/email.html]: `Position` model doesn't have the `catalog_name` property and `mark` field too. Let's create a todo for this bug

File: templates/search/items_list.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
     </td>
     <td class="table-td table-name">
       <a class="table-link" href="{{ item.get_absolute_url }}">
-        {{ item.name }} {{ item.mark }} {{ item.specification }}
```
Comment (artemiy312) [inline templates/search/items_list.html]: You missed `specification` here

File: templates/ecommerce/prices/price.yml (hunk 1 above)
Comment (duker33) [reply templates/ecommerce/prices/price.yml]: @artemiy312 , `option.catalog` name is `{{ option.product.name }} {{ option.mark }}`.
It's not equivalent pattern, so we can't use it here

File: templates/ecommerce/order/email.html (hunk 2 above)
Comment (duker33) [reply templates/ecommerce/order/email.html]: it's not about stb repo, so created it manually
https://github.com/fidals/refarm-site/issues/322

File: templates/search/items_list.html (hunk 3 above)
Comment (duker33) [reply templates/search/items_list.html]: @artemiy312 , now item has no specification field.
Series feature will hadle specification field content in more correct way

File: templates/ecommerce/order/email.html (hunk 2 above)
Comment (artemiy312) [reply templates/ecommerce/order/email.html]: @duker33 Right now `Product.mark` exist only in STB, but not in SE, so why not about this repo?

Comment (duker33) [reply templates/ecommerce/order/email.html]: @artemiy312 , sorry, i'll create pdd subtask

**Pull request merged**